DecipherNow/gm-fabric-dashboard
# fix: 1001 add pseudo element to expand clickable area

Resolves #1001 

## Proposed Changes

  -  the clickable area of document icon moves on hover/active.  i gave the docs icon a pseudo element that expands the clickable area,  making it easier to click on transition/transform. 
  
## How to Test

  - Open Safari 10.1 and try clicking on edges of doc icon, and it should open document link with one click
